test/org/sagebionetworks/bridge/play/controllers/StudyControllerTest.java

```diff
@@ -201,7 +202,30 @@ public void canGetUploadsForStudy() throws Exception {
         assertEquals(startTime, retrieved.getStartTime());
         assertEquals(endTime, retrieved.getEndTime());
     }
-    
+
+    @Test
+    public void canGetUploadsForSpecifiedStudy() throws Exception {
+        doReturn(mockSession).when(controller).getAuthenticatedSession(WORKER);
+
+        DateTime startTime = DateTime.parse("2010-01-01T00:00:00.000Z");
+        DateTime endTime = DateTime.parse("2010-01-02T00:00:00.000Z");
+
+        DateTimeRangeResourceList<? extends Upload> uploads = new DateTimeRangeResourceList<>(Lists.newArrayList(),
+                startTime, endTime);
+        doReturn(uploads).when(mockUploadService).getStudyUploads(studyId, startTime, endTime);
+
+        Result result = controller.getUploadsForStudy(studyId.getIdentifier(), startTime.toString(), endTime.toString());
+        assertEquals(200, result.status());
+
+        verify(mockUploadService).getStudyUploads(studyId, startTime, endTime);
+
+        // in other words, it's the object we mocked out from the service, we were returned the value.
+        DateTimeRangeResourceList<? extends Upload> retrieved = BridgeObjectMapper.get()
+                .readValue(Helpers.contentAsString(result), UPLOADS_REF);
+        assertEquals(startTime, retrieved.getStartTime());
+        assertEquals(endTime, retrieved.getEndTime());
+    }
```

Comment:
Add tests to show that the right exception is thrown if not a worker? Verify if the studyId is not correct, the correct exception is thrown.